zerver/webhooks/taiga/view.py

```diff
@@ -30,10 +30,11 @@
 from zerver.lib.webhooks.common import check_send_webhook_message
 from zerver.models import UserProfile
 
+
 @api_key_only_webhook_view('Taiga')
 @has_request_variables
 def api_taiga_webhook(request: HttpRequest, user_profile: UserProfile,
-                      message: Dict[str, Any]=REQ(argument_type='body')) -> HttpResponse:
+                      message: Dict[str, Any] = REQ(argument_type = 'body')) -> HttpResponse:
```

Comment:
In function arguments, there should be no spaces around `=`, so this needs to be reversed.